src/features/dashboard/components/SituationSetupModal.tsx

```
@@ -0,0 +1,60 @@
+import { Button } from "@/components/ui/button";
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@/components/ui/dialog";
+import { Input } from "@/components/ui/input";
+import { Label } from "@radix-ui/react-label";
```

ESLint

[error] 'Label' is defined but never used.

(@typescript-eslint/no-unused-vars)